core/services/workflows/engine.go

```diff
@@ -93,6 +107,7 @@ LOOP:
 					return fmt.Errorf("failed to get capability with ref %s: %s, retrying in %d seconds", n.Type, innerErr, retrySec)
 				}
 
+				// We only support CallbackExecutable capabilities for now 
```

Comment:
More accurately: "Triggers, Consensus and Targets are all `CallbackExecutable` capabilities.".